lobster/tools/core/html_report.py

```diff
@@ -465,6 +465,9 @@ def main():
                     help="path to dot utility (https://graphviz.org), \
                     by default expected in PATH",
                     default=None)
+    ap.add_argument("--color_blind",
+                        action="store_true",
+                        help="Enable color blind mode")
```

Comment:
be more specific in the help message: 
something like "uses a color palette with a higher contrast"